### PR TITLE
Releae Note for ONE 1.11.0

### PR DESCRIPTION
**Pull request by chunseoklee**
This is the release note for ONE 1.11.0

### CODE REVIEW

**Comment by chunseoklee** [comment]
@Samsung/nnfw Please add release candidates as comment.

**Comment by seanshpark** [comment]
## ONE Compiler

### Changes
- `one-build` command added as representative command
- one-cmds are now revised to python script and supports configuration file as input parameters
- added more optimization pass in `one-optimize`; `fuse_preactivation_batchnrom`, `make_batchnorm_gamma_positive` and `fuse_activation_function`

**Comment by hseok-oh** [comment]
### Runtime backend operation support

- CPU backend
  - float: AddN, Floor, UniDirectionalSequenceLSTM
  - uint8: Dequantize, Rank
  - int8: Dequantize, Rank, Shape

**Review by mhs4670go — APPROVED** [review]
LGTM

**Review by hseok-oh — APPROVED** [review]
LGTM

File: docs/release/1.11/release-note-1.11.0.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+## ONE Compiler
+
+### Compiler supports more operations
+- MaxPoolWithArgMax by CustomOp
+
+### Changes 
+- `one-build` command added as representative command
+- one-cmds are now revised to python script and supports configuration file as input parameters
+- added `rawdata2hdf5` tool to prepare quantization data
+- added more optimization pass in `one-optimize`; `fuse_preactivation_batchnrom`, `make_batchnorm_gamma_positive` and `fuse_activation_function`
```
Comment (jinevening) [inline docs/release/1.11/release-note-1.11.0.md:12]:
```suggestion
- added more optimization passes in `one-optimize`; `fuse_preactivation_batchnorm`, `make_batchnorm_gamma_positive` and `fuse_activation_function`
```

File: docs/release/1.11/release-note-1.11.0.md [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+
+## ONE Compiler
+
+### Compiler supports more operations
+- MaxPoolWithArgMax by CustomOp
+
+### Changes 
+- `one-build` command added as representative command
+- one-cmds are now revised to python script and supports configuration file as input parameters
+- added `rawdata2hdf5` tool to prepare quantization data
```
Comment (jinevening) [inline docs/release/1.11/release-note-1.11.0.md:11]:
```suggestion
- added `rawdata2hdf5` tool to help creating input datasets for calibration
```

**Review by hseok-oh — APPROVED** [review]
LGTM

**Review by mhs4670go — APPROVED** [review]
LGTM

**Review by jinevening — APPROVED** [review]
LGTM

**Pull request merged**